test/rng_testsuite/extra_methods_tests/distributions_methods.pass.cpp

```diff
@@ -0,0 +1,269 @@
+// -*- C++ -*-
+//===-- engines_methods.hpp ----------------------             -------------===//

```

Comment:
This is not aligned with the name of the test and contains extra spaces. Maybe it's better to not mention name of tests in tests' code?

Reply:
changed the name and removed extra spaces, I guess it's better to leave the filename in file description because we have it in other tests